modules/local/antismash_summary.nf

```diff
@@ -0,0 +1,37 @@
+process ANTISMASH_SUMMARY {
+    tag "${meta.id}"
+    label 'process_low'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mgnify-pipelines-toolkit:1.0.5':
+        'biocontainers/mgnify-pipelines-toolkit:1.0.5' }"
```

Comment:
it's not a perfect solution, but one thing the amplicon-pipeline has for keeping toolkit versions up to date is this: https://github.com/EBI-Metagenomics/amplicon-pipeline/blob/857d283a0cc8a7e54ff59faa31d8b06a912f3184/modules/local/classify_var_regions/main.nf#L6-L8

you would just add the current version to the nextflow.config as a param: https://github.com/EBI-Metagenomics/amplicon-pipeline/blob/857d283a0cc8a7e54ff59faa31d8b06a912f3184/nextflow.config#L57

not needed for this PR specifically but just in general for the pipeline 😄

Reply:
that is perfect! I will do it for all mpt tools in ASA! thank you!